### ISSUE TITLE
Error metric

### ISSUE THREAD

**Issue by davidggz** [brijeshiitg/Pytorch-implementation-of-SRNet#9]
Thank you for your implementation in Pytorch, it's been quite useful to compare with my implementation. I'm trying to replicate the SRNet results that are shown in its paper but I haven't been able yet. The thing is that, even using the data set that they claim the use in their paper, I'm not able to improve the validation accuracy from 0.5. I've been wondering what could be the error and maybe it's related to the fact that I'm using binary crossentropy as my loss function, even though in the paper they claim the use "total classification error probability". The question is, did you use any special error function or you just used the typical like binary crossentropy?

**Comment by davidggz** [comment]
I saw in other papers that they use categorical or binary crossentropy depending on the number of output neurons. The thing is that I am still not able to improve the validation from 0.5 even though I think I replicated perfectly the model.
